### ISSUE TITLE
Combine Detection Pval and Avg Signal results

### ISSUE THREAD

**Issue by countdigi** [USF-HII/BeadArray#4]
@parikhhm and I had a discussion and we have a new requirement with is a variation on https://github.com/USF-HII/BeadArray#combine-expression

---

We want to merge all the files with *_Detection_Pval.txt

For example, 9985134046_Detection_Pval.txt file (/home/h/hparikh/tmp/BeadArray-non-donor/tmp/work/method-c-step-1-non-donor/chip/9985134046/results)
And 9985134050_Detection_Pval.txt (/home/h/hparikh/tmp/BeadArray-non-donor/tmp/work/method-c-step-1-non-donor/chip/9985134050/results).

Basically adding “ID” tag on first raw of first column and then use all the values in first column as primary key for merging all the *_Detection_Pval.txt files from all different beadchips.

----
Here is an example of 3 inputs and the desired output:

[Input_Merge_Files.xlsx](https://github.com/USF-HII/BeadArray/files/298917/Input_Merge_Files.xlsx)

**Comment by countdigi** [comment]
Code added to handle both cases in commits: d6e4f1a, 446b5cf, f2d94ab, and 3638810

**Comment by countdigi** [comment]
cc/ @ksungithub who hacked it out w/ me :-)

**Comment by ksungithub** [comment]
I was just an onlooker haha :D
________________________________
